guidedmodules/migrations/0055_auto_20210121_2312.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.0.11 on 2021-01-21 23:12
```

Comment:
Need to only have one migration file that is affecting app version instead of 3.

Reply:
Fixed! Good catch